### ISSUE TITLE
Behavior of the case of generating a document from other than request-specs.

### ISSUE THREAD

**Issue by joe-re** [r7kamura/autodoc#25]
I tryed to generate a doc from other than request-specs.

In this case, Autodoc succeeded in generating a doc, but it didn't have intended filepath.

I think, it should generate a doc from only residing spec/requests.

How do you think?

### spec
```bash
$ cat spec/controllers/api/v1/workbooks_controller_spec.rb
```

```ruby
require 'rails_helper'

RSpec.describe Api::V1::WorkbooksController, type: :controller do
  render_views

  describe '#index' do
    context 'has workbooks' do
      before do
        @workbook = create(:workbook)
        get :index, format: :json
      end
      it 'get workbook list', autodoc: true do
        expect(response.status).to eq 200
      end
    end
  end
end
```

### result
```bash
$ AUTODOC=1 bundle exec rspec
```

```bash
$ tree doc
doc
└── spec
    └── controllers
        └── api
            └── v1
                └── workbooks_controller_spec.rb
```

**Comment by r7kamura** [comment]
I hadn't thought about such kind of use of using :autodoc other than request-specs, however, some people might benefit from that. I think modifying [autoload/document.rb](https://github.com/r7kamura/autodoc/blob/e45b08c9d34e90ec6d27de2bf29b58ebc01202e7/lib/autodoc/document.rb#L21) can fix the filepath problem.

**Comment by joe-re** [comment]
I see. How is this? #26

**Comment by r7kamura** [comment]
Released a new version v0.4.5 that includes #26 :sparkles: :gem: :sparkles:
